workflow/signals.py

```diff
@@ -195,7 +196,7 @@ def check_seats_save_user_groups(sender, instance, **kwargs):
 # ORGANIZATION SIGNALS
 @receiver(signals.post_save, sender=Organization)
 def sync_save_track_organization(sender, instance, **kwargs):
-    if os.getenv('APP_BRANCH'):
+    if os.getenv('APP_BRANCH') and settings.TOLA_TRACK_SYNC_ENABLED:
```

Comment:
If we add the `TOLA_TRACK_SYNC_ENABLED` env var, we don't need to check the `APP_BRANCH` anymore. We were checking it to execute the sync and as we have a new env var, you can remove `os.getenv('APP_BRANCH')`.